CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 * Fixed an issue when feedback UI was always appearing for short routes ([#2871](https://github.com/mapbox/mapbox-navigation-ios/pull/2871))
 * Fixed automatic day/night switching ([#2881](https://github.com/mapbox/mapbox-navigation-ios/pull/2881))
+* Fixed an issue where presenting `NavigationViewController` could sometimes interefere with view presentation in other windows ([#2897](https://github.com/mapbox/mapbox-navigation-ios/pull/2897))
```

Comment:
Since StyleManagerDelegate is a public protocol, we should also mention the new addition on top of mentioning this bug fix:

```suggestion
* Fixed an issue where presenting `NavigationViewController` could sometimes interfere with view presentation in other windows. ([#2897](https://github.com/mapbox/mapbox-navigation-ios/pull/2897))
* Added the `StyleManagerDelegate.styleManager(_:viewForApplying:)` method to determing which part of the view hierarchy is affected by a change to a different `Style`. ([#2897](https://github.com/mapbox/mapbox-navigation-ios/pull/2897))
```